docs/app/Examples/elements/Container/Types/ContainerTextExample.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+
+import React, { Component } from 'react'
+import { Container, Message } from 'stardust'
+
+export default class ContainerContainerExample extends Component {
```

Comment:
Done

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 16:58
>
> In docs/app/Examples/elements/Container/Types/ContainerTextExample.js 
> <https://github.com/TechnologyAdvice/stardust/pull/277#discussion_r68419885>:
>
> > @@ -0,0 +1,21 @@
> > +/* eslint-disable max-len */
> > +
> > +import React, { Component } from 'react'
> > +import { Container, Message } from 'stardust'
> > +
> > +export default class ContainerContainerExample extends Component {
> -class ContainerContainerExample
> +class ContainertextExample
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/277/files/06fde8b6601dcbdb89260c966a8eba1a11f55df0#r68419885>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHs-APioSkl5c_w7RXiAMgUdImf-P9ks5qO_6mgaJpZM4I9hiP>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>